### PR TITLE
skip osg-version stuff for osg-3.5 (SOFTWARE-2915)

### PR DESCRIPTION
**Pull request by edquist**

### CODE REVIEW

**Review by brianhlin — APPROVED** [review]
LGTM

File: 0-generate-pkg-list [hunk 1]
```diff
@@ -25,7 +25,11 @@ detect_rescue_file
 # Promote osg-version to testing #
 ##################################
 promote_osg_version() {
-    for ver in ${versions[@]/upcoming/}; do # We don't build osg-version for upcoming
+    for ver in ${versions[@]}; do
+        case $ver in
+            # We don't build osg-version for upcoming or 3.5+
+            3.5 | upcoming ) continue ;;
```
Comment (brianhlin) [inline 0-generate-pkg-list]: I believe `$ver` is the full version string here, e.g. `3.4.34`. Does this actually work for that?

**Review by brianhlin — COMMENTED** [review]
Actually, one question

File: 0-generate-pkg-list (hunk 1 above)
Comment (edquist) [reply 0-generate-pkg-list]: herm... let's see...

Comment (edquist) [reply 0-generate-pkg-list]: No, you're right it won't work - i was mistaken about the format of `$ver` - i'll fix.

Comment (edquist) [reply 0-generate-pkg-list]: Globs are allowed in the `case` patterns, so I check for `3.5.*` instead.

**Review by brianhlin — APPROVED** [review]
LGTM

**Review by timtheisen — APPROVED** [review]
Looks good

**Pull request merged**